server/pkg/stellar/client.go

```diff
@@ -101,6 +119,20 @@ func (c *Client) Load(ctx context.Context, conState jsonrpc.State, args Load) er
 	return nil
 }
 
+func (c *Client) CreateAccount(ctx context.Context, conState jsonrpc.State, network string) (string, error) {
+	cl, err := stellargoclient.NewClient("", network)
```

Comment:
Now this is pretty confusing. You call `CreateAccount` , get a seed returned and the loaded  stellarclient gets swapped with a new one.

Reply:
This call is more meant to be called as the first call in a session. How should I do it differently? Allow creating the account and not swap the old one?

Reply:
Ok, get it now, this was not clear to me. I think some code comment here would be useful.
Maybe also extend manual/src/server/functionality/stellar.md ,the comment on the vclient create_account function and manual/src/v-clients/stellar/stellar.md